skills/ai-first-documentation/scripts/audit-docs-mechanical.ps1

```
@@ -0,0 +1,561 @@
+            if ($headingText -match '^([A-Z][0-9]+):\s+(\S+)') {
+                $checkId = $Matches[1]
+                $filePath = $Matches[2].Trim()
+                # Normalize separators for key matching
+                $normalizedPath = $filePath -replace '\\', '/'
+                $key = "${checkId}:${normalizedPath}"
+                $map[$key] = $headingText
+            }
+        }
+    } catch {
+        # If decision record can't be read, skip waiver logic (proceed with mechanical verdict)
+    }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)